io.gitlab.hamadmarri.gamma.json

```diff
@@ -0,0 +1,62 @@
+{
+	"app-id": "io.gitlab.hamadmarri.gamma",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.34",
+	"sdk": "org.gnome.Sdk",
+	"command": "gamma",
+	"finish-args": [
+		"--share=network",
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=ipc",
+		"--talk-name=org.freedesktop.DBus.Hello",
+		"--talk-name=org.a11y.*",
+		"--talk-name=org.freedesktop.DBus.AddMatch",
+		"--talk-name=org.freedesktop.DBus.StartServiceByName",
+		"--talk-name=org.freedesktop.DBus.GetNameOwner",
+		"--talk-name=org.gtk.vfs.Daemon.ListMonitorImplementations",
+		"--talk-name=org.freedesktop.DBus.RemoveMatch",
+		"--talk-name=org.freedesktop.DBus.RequestName",
+		"--talk-name=org.freedesktop.DBus.GetId",
+		"--talk-name=org.freedesktop.DBus.Properties.GetAll",
+		"--talk-name=org.freedesktop.DBus.Peer.Ping",
+		"--talk-name=org.gtk.Actions.DescribeAll",
+		"--talk-name=org.gtk.Private.RemoteVolumeMonitor.IsSupported",
+		"--talk-name=org.gtk.Private.RemoteVolumeMonitor.List",
+		"--talk-name=org.gtk.vfs.MountTracker.ListMounts2",
+		"--talk-name=org.gtk.vfs.MountTracker.LookupMount",
+		"--talk-name=org.gtk.vfs.Daemon.GetConnection",
+		"--talk-name=org.freedesktop.DBus.Error.ServiceUnknown",
+		"--filesystem=host",
+		"--own-name=io.gitlab.hamadmarri.gamma"
+	],
+	"modules": [
+		{
+			"name": "gtksourceview",
```

Comment:
You should also add `cleanup` array to delete development headers and other unneeded files.

Reply:
do you mean the gtksource files?